### ISSUE TITLE
Get-NSVolumeCollection Error

### ISSUE THREAD

**Issue by wild0104** [jrich523/NimblePowerShell#7]
When trying to run the Get-NSVolumeCollection I get the following error:

Exception calling "getProtPolList" with "2" argument(s): "There is an error in XML document (2, 9459)."
At C:\Users\jacob.wilde\Documents\WindowsPowerShell\Modules\Nimble\VolCol.ps1:37 char:9
+         $rtncode = $Script:NSUnit.getProtPolList($sid.Value, [ref]$ppl)
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : InvalidOperationException

Get-NSVolumeCollection : Error getting volume collection list! code:
At line:1 char:1
+ Get-NSVolumeCollection
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Write-Error], WriteErrorException
    + FullyQualifiedErrorId : Microsoft.PowerShell.Commands.WriteErrorException,Get-NSVolumeCollection

I tried passing it the name of a Volume Collection but get the same error.

**Comment by jrich523** [comment]
What firmware version are you running?

I'm working on a v2 version right now (there is a branch for it) that I believe has that updated already

**Comment by wild0104** [comment]
Firmware on the Nimble array is 2.1.4.

I saw the v2 branch, but the install.ps1 script still looks to pull the master branch files and install those.

**Comment by jrich523** [comment]
Yeah you can just pull the branch and put the files in a folder called Nimble, and either replace the Nimble folder you have now in your profile (my documents\windowspowershell\modules)

or you can put it anywhere, and then just do Import-Module .\Nimble     #change to whatever the path of the new nimble folder is

**Comment by wild0104** [comment]
Tried that, still throws the same error, can't even run Get-NSVolume without it. I was trying to script creating clones off a volume collection snapshot, but it looks like with either version i'll have to create the clone off each volume manually so i'll just work with the master branch for now.

@jrich523 thanks for the great PowerShell tools for Nimble!

**Comment by jrich523** [comment]
Ahh that's what I'm working on now, but I'm having some issues with the api. Been fairly busy lately so haven't had time to work on it but that should change soon. I'll mark this resolved when I get it
